packages/dcos-launch/extra/README.md

```diff
@@ -0,0 +1,28 @@
+# dcos-launch User Guide
+## Purpose
+`dcos-launch` is intendends to provide the following:
+* Consistent Interface: launching on Azure or AWS can involve finding a suitable client API and then sifting through hundreds of specific API calls to find the combination that will allow deploying DC/OS. `dcos-launch` has the exact methods required to take a DC/OS build artifact and deploy it to an arbitrary provider with no other tools.
```

Comment:
Explaining the background, and features of dcos-launch in the introduction seems a bit unnecessary to me.

When a reader is looking for `dcos-launch` and lands on the readme.md, perhaps he will need a high level introduction like

`dcos-launch provides a consistent interface to launch dcos on different infrastructure providers`

and then look for the sub-commands.

And if there is argument on why "dcos-launch" is required (or better than something else), you might consider adding these rationale statements at the end of the guide.

What do you think about this @mellenburg ?